src/libraries/Common/src/Interop/OSX/Interop.SystemConfiguration.cs

```diff
@@ -18,10 +18,10 @@ internal static partial class SystemConfiguration
         internal struct SCDynamicStoreContext
         {
             public CFIndex version;
-            public IntPtr Info;
-            public IntPtr RetainFunc;
-            public IntPtr ReleaseFunc;
-            public CFStringRef CopyDescriptionFunc;
+            public IntPtr info;
```

Comment:
Fixed this structure to follow interop coding conventions (use the same name and casing as the Apple SDK definition).